src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/SplitNCigarReads.java

```diff
@@ -200,7 +252,26 @@ private static GATKRead splitReadBasedOnCigar(final GATKRead read, final int cig
             forSplitPositions.addSplicePosition(contig, splitStart, splitEnd);
         }
 
-        return ReadClipper.hardClipToRegionIncludingClippedBases(read, startRefIndex, stopRefIndex);
+        return ReadClipper.softClipToRegionIncludingClippedBases(read,startRefIndex,stopRefIndex);
     }
 
+    /**
+     * A method that repairs the NH and NM tags for a group of reads
+     *
+     * @param readFamily         the a list of reads where the first item is the read to be marked as primary
+     * @param header             the file header to associate with the given reads
+     * @return a non-null read representing the section of the original read being split out
+     */
+    public static void repairSuplementaryTags(List<GATKRead> readFamily, SAMFileHeader header) {
+        for (GATKRead read : readFamily) {
+            if (readFamily.size()>1) {
```

Comment:
weirrd spacing here, in general you'd want spaces around `>` like you did below.